trace-sdk/src/main/java/io/bitrise/trace/configuration/ConfigurationManager.java

```diff
@@ -112,8 +114,14 @@ public static synchronized void init(@NonNull final Context context) {
     getInstance();
     initialised = true;
     configurationMap = new HashMap<>();
-    importConfigurationFromBuildConfig(context);
-    importConfigurationFromResValues(context);
+
+    try {
```

Comment:
this regularly causes crashes when unit/integration testing without the token set - i don't think these should ever crash anything - but should definitely show an error instead. What do you think?

Reply:
Makes sense.